src/main/resources/resource/OpenNi/OpenNi.py

```diff
@@ -6,4 +6,20 @@
 # very minimal script to start
 
 openni = Runtime.start("openni", "OpenNi")
+python = Runtime.getService("python")
+
```

Comment:
Runtime.start('python', 'Python') is always safe to call .. if it exists it just gets it ... if it doesn't it creates it

Reply:
implicitly, the python service has already been started, this is more to serve as an example that you can call getService to get a service by it's name if it's already been started.